src/stepFunctions/commands/visualizeStateMachine/aslVisualizationCDKManager.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+import { AslVisualizationCDK } from './aslVisualizationCDK'
+import { AbstractAslVisualizationManager } from './abstractAslVisualizationManager'
+import { ConstructNode } from '../../../cdk/explorer/nodes/constructNode'
+import { getLogger } from '../../../shared/logger'
+import { StateMachineGraphCache } from '../../utils'
+
+export class AslVisualizationCDKManager extends AbstractAslVisualizationManager {
+    protected readonly name: string = 'AslVisualizationCDKManager'
+    protected readonly managedVisualizations: Map<string, Map<string, AslVisualizationCDK>> = new Map<
+        string,
+        Map<string, AslVisualizationCDK>
+    >()
+
+    public constructor(extensionContext: vscode.ExtensionContext) {
+        super(extensionContext)
+    }
+
+    //vscode.Memento passed in to update StateMachineGraphCache on each visualization
+    public async visualizeStateMachine(
+        globalStorage: vscode.Memento,
+        node: ConstructNode
+    ): Promise<vscode.WebviewPanel | undefined> {
+        const logger = getLogger()
+        const cache = new StateMachineGraphCache()
+        const cdkOutPath = node.id.replace(`/tree.json/${node.tooltip}`, ``)
+        const workspaceName = this.getCDKAppWorkspaceName(cdkOutPath)
+        const existingVisualization = this.getExistingVisualization(workspaceName, node.tooltip)
+        const stateMachineName = node.label
+        const appName = node.tooltip.replace(`/${stateMachineName}`, ``)
+        const templatePath = String(cdkOutPath) + `/${appName}.template.json`
```

Comment:
Why do you need the `String()` declaration? Also, don't use `+` to concatenate when you're already using a template literal.
```suggestion
        const templatePath = `${cdkOutPath}/${appName}.template.json`
```

Also: __if you're using these paths here and elsewhere to interact with the filesystem__, make sure that this and other instances of `/` are handled correctly in Windows. You can either use `pathutils.normalize` or if you want to write these by hand, use `path.sep` in place of `/`